package.json

```diff
@@ -11,13 +11,30 @@
     "emoji",
     "react"
   ],
-  "author": "Arron Hunt",
+  "author": "Amos Tan",
   "license": "MIT",
-  "dependencies": {
-    "emoji-datasource": "^6.0.0"
+  "peerDependencies": {
```

Comment:
go with normal `dependencies`, `peerDependencies` is a bit weird for consuming, normal dependencies should be fine for this case.